uui-components/src/buttons/ButtonBase.tsx

```diff
@@ -60,6 +60,8 @@ export abstract class ButtonBase<ButtonProps extends ButtonBaseProps> extends Re
         return this.props.tabIndex || 0;
     }
 
+    getAriaAttrs?(): Record<`aria-${string}`, string | boolean>;
```

Comment:
Probably we can make this functions not specific for Aria attributes, that we can pass any Button attributes if needed, it will be more versatile.